### PR TITLE
Remove mod chooser reference from music installation prompt.

### PR DESCRIPTION
**Pull request by pchote**

### CODE REVIEW

File: mods/common/chrome/lobby-music.yaml [hunk 1]
```diff
@@ -160,27 +160,21 @@ Container@LOBBY_MUSIC_BIN:
...
-					Align: Center
-					Text: from the "Manage Content" menu
-				Label@DESCC:
-					Y: 125
+					Y: 115
 					Width: PARENT_RIGHT - 24
 					Height: 25
 					Align: Center
-					Text: in the mod chooser.
+					Text: from the "Manage Content" menu
```
Comment (abcdefg30) [inline mods/common/chrome/lobby-music.yaml]: You're missing the dot here at the end (you added it everywhere else).

**Review by rob-v — APPROVED** [review]
👍  (after adding missing dot)

**Comment by pchote** [comment]
Fixed.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/acd7454f571d1e0fbfe97c9f6742eba79a35e9f3...8ef9225d8647009f42ca74a03aa6f83d7db05f27)

**Pull request merged**